qiskit/providers/ibmq/job/queueinfo.py

```diff
@@ -28,7 +28,17 @@
 
 @bind_schema(InfoQueueResponseSchema)
 class QueueInfo(BaseModel):
-    """Queue information related to a job."""
+    """Queue information for a job.
+
+    Attributes:
```

Comment:
Since the docstrings inside `__init__.py` explain the parameters, is this necessary?

<img width="749" alt="image" src="https://user-images.githubusercontent.com/11485594/74569710-0d25a800-4f49-11ea-8ce7-8ec3f3c4d7a8.png">

Reply:
Those are actually public attributes (the parameters just happen to have the same name). I couldn't find a way to list them under "attributes", so if you do please let me know!